get_together/templates/get_together/emails/reminder.txt

```diff
@@ -0,0 +1,11 @@
+{% block content %}
+== Event Reminder Notification ==
+
+You have an event coming up!
+
+Name: {{ event.name|striptags }}
+Time: {{event.local_start_time}}
+Location: {{ event.location|striptags }}
+
+Event page: {{event.get_absolute_url}}
```

Comment:
Use event.get_full_url instead of get_absolute_url here, because it includes the https://<site.domain>/ part of a URL, while get_absolute_url only returns the absolute path.